deployment/terraform/modules/osv/main.tf

```diff
@@ -120,6 +125,17 @@ resource "google_storage_bucket" "debian_osv_conversion_bucket" {
   }
 }
 
+resource "google_storage_bucket" "logs_bucket" {
+  project                     = var.project_id
+  name                        = var.logs_bucket
+  location                    = "US"
+  uniform_bucket_level_access = true
+
+  lifecycle {
+    prevent_destroy = true
+  }
+}
+
```

Comment:
It looks like `osv-logs` and `osv-test-logs` both already exist (but are empty?) on their respective projects.
I don't think terraform is smart enough to automatically pick up that they already exist, the `terraform plan` output says it's trying to create these.

Since they're currently empty, could you delete it on `oss-vdb-test` before merging, and delete it on `oss-vdb` before running `terraform apply` locally?

Reply:
Oh, I had just created them (and manually redid the bucket configuration). My intention here was to make a future `terraform apply` a no-op...

Reply:
I think it'd generally be better to let terraform create them from scratch, but if you want a no-op you could `terraform import` them:
```sh
# in environments/oss-vdb-test
terraform import module.osv_test.google_storage_bucket.logs_bucket oss-vdb-test/osv-test-logs

# in environments/oss-vdb
terraform import module.osv.google_storage_bucket.logs_bucket oss-vdb/osv-logs
```

I'd re-run the `terraform-plan` check if you do that, to make sure nothing is changing.

Reply:
Okay I'll do as you suggested and let terraform (re)create the bucket by deleting it first.